openshift/pg_tileserv/nginx.conf

```diff
@@ -0,0 +1,150 @@
+    gzip_http_version  1.0;
+
+    # Compression level (1-9).
+    # 5 is a perfect compromise between size and cpu usage, offering about
+    # 75% reduction for most ascii files (almost identical to level 9).
+    gzip_comp_level    5;
+
+    # Don't compress anything that's already small and unlikely to shrink much
+    # if at all (the default is 20 bytes, which is bad as that usually leads to
+    # larger files after gzipping).
+    gzip_min_length    256;
+
+    # Compress data even for clients that are connecting to us via proxies,
+    # identified by the "Via" header (required for CloudFront).
+    gzip_proxied       any;
+
+    # Tell proxies to cache both the gzipped and regular version of a resource
+    # whenever the client's Accept-Encoding capabilities header varies;
+    # Avoids the issue where a non-gzip capable client (which is extremely rare
+    # today) would display gibberish if their proxy gave them the gzipped version.
+    gzip_vary          on;
+
+    # This assigns the number and the size of the compression buffers.
+    # The default is gzip_buffers 4 4k;
+    # Increase this value to make sure that big Javascript or
+    # CSS files can be compressed as well.
+    gzip_buffers 16 8k;
+
+    # Compress all output labeled with one of the following MIME-types.
+    gzip_types
+      application/atom+xml
+      application/javascript
+      application/x-javascript
+      text/xml
+      application/xml+rss
+      text/javascript
+      application/json
+      application/rss+xml
+      application/vnd.ms-fontobject
+      application/x-font-ttf
+      application/x-web-app-manifest+json
+      application/xhtml+xml
+      application/xml
+      application/vnd.mapbox-vector-tile
+      font/opentype
+      image/svg+xml
+      image/x-icon
+      text/css
+      text/plain
+      text/x-component;
+    # text/html is always compressed by HttpGzipModule
+
+    include       /etc/opt/rh/rh-nginx112/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    # Load modular configuration files from the /etc/nginx/conf.d directory.
+    # See http://nginx.org/en/docs/ngx_core_module.html#include
+    # for more information.
+    include /opt/app-root/etc/nginx.d/*.conf;
+
+
+    proxy_cache_path /tmp/cache levels=1:2 keys_zone=tile_cache:10m max_size=1g 
+                    inactive=30m use_temp_path=off;
+
+    server {
+        listen       8080 default_server;
+        listen       [::]:8080 default_server;
+        server_name  _;
+        root         /opt/app-root/src;
+
+        # Load configuration files for the default server block.
+        include      /opt/app-root/etc/nginx.default.d/*.conf;
+
+        location /gwells/tiles/ {
+            proxy_pass http://localhost:7800/;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+
+            proxy_cache tile_cache;
+            proxy_cache_valid any 30m;
```

Comment:
I think this cache time should be shorter until there is explicit cache invalidation.